integreat_cms/cms/forms/events/event_form.py

```diff
@@ -1,6 +1,7 @@
 import logging
 
 from datetime import time
+import datetime
```

Comment:
```suggestion
from datetime import time, timedelta
```
We can combine these imports